p2p/libp2p/messagesHandler.go

```diff
@@ -25,6 +28,7 @@ var messageHeader = 64 * 1024 // 64kB
 var maxSendBuffSize = (1 << 21) - messageHeader
 
 const durationBetweenSends = time.Microsecond * 10
+const equivalentMessagesCacheSize = 1000
```

Comment:
maybe extract this into config

Reply:
I will defer this change to a later update when we need to reconfigure it.